style/amenity-points.mss

```diff
@@ -2281,12 +2282,13 @@
         text-fill: @tourism;
         text-face-name: @bold-fonts; /*rendered bold to improve visibility since theme parks tend to have crowded backgrounds*/
       }
-      [feature = 'amenity_kindergarten'],
       [feature = 'amenity_school'],
-      [feature = 'amenity_college'],
-      [feature = 'amenity_university'] {
-        text-fill: darken(@societal_amenities, 80%);
-      }
+        [feature = 'amenity_kindergarten'],
+        [feature = 'amenity_college'],
+        [feature = 'amenity_research_institute'],
+        [feature = 'amenity_university'] {
```

Comment:
You changed the order of the lines (kindergarten, school are swapped and the new entry is between collage and university). They are now different from the list above. Best to stay in one order.

Reply:
@LaoshuBaby could you address the comment above?